__tests__/__snapshots__/About.snapshot.tsx.snap

```diff
@@ -0,0 +1,168 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Component AboutModal - test AboutModal - snapshot 1`] = `
+<RCTSafeAreaView
+  emulateUnlessSupported={true}
+  style={
+    Object {
+      "alignItems": "stretch",
+      "backgroundColor": "rgb(242, 242, 242)",
+      "display": "flex",
+      "height": "100%",
+      "justifyContent": "flex-start",
+    }
+  }
+>
+  <View>
+    <View
+      style={
+        Object {
+          "alignItems": "center",
+          "backgroundColor": "rgb(255, 255, 255)",
+          "paddingBottom": 25,
+          "paddingTop": 25,
+        }
+      }
+    >
+      <Text
+        style={
+          Object {
+            "color": "rgb(28, 28, 30)",
+            "fontSize": 28,
+            "marginTop": 5,
+            "padding": 5,
+          }
+        }
+      >
+        zecwallet DAO v1.7.8
+      </Text>
+    </View>
+    <View
+      style={
+        Object {
+          "alignItems": "center",
+          "display": "flex",
+          "marginTop": -25,
+        }
+      }
+    >
+      <Image
+        source={
+          Object {
+            "testUri": "../../../assets/img/logobig.png",
+          }
+        }
+        style={
+          Object {
+            "height": 50,
+            "resizeMode": "contain",
+            "width": 50,
+          }
+        }
+      />
+    </View>
+  </View>
+  <RCTScrollView
+    contentContainerStyle={
+      Object {
+        "alignItems": "stretch",
+        "flexDirection": "column",
+        "justifyContent": "flex-start",
+      }
+    }
+    style={
+      Object {
+        "maxHeight": "85%",
+      }
+    }
+  >
+    <View>
+      <Text
+        style={
+          Object {
+            "color": "rgb(28, 28, 30)",
+            "opacity": 0.65,
+          }
+        }
+      >
+        
+
+        Copyright (c) 2018-2021, Aditya Kulkarni.
+        
+
+        
+
+        Built with React Native.
+        
+
+        
+
+        The MIT License (MIT) Copyright (c) 2018-2021 Zecwallet
```

Comment:
Maybe the "zingolabs developers (c) 2022" should be a thing?

Reply:
It would be a good idea to change it for that... if we are sure about 'copyright' thingy.

Reply:
But this text of About screen, hasn't nothing to do with this PR and CI